hackerrank/simple-array-sum/challenge.test.js

```diff
@@ -0,0 +1,9 @@
+const simpleArraySum = require('./challenge');
+
+test('adds 1 + 2 + 3 + 4 + 10 + 11  to equal 31', () => {
```

Comment:
This test description is not good enough, you can't list all the array items like this.
Can we say something like "sums an array for integers" and for a larger one it can be  "sums a larger array for integers"